### DISCUSSION TITLE
[Feature] Multi IP Access for Mobil App

### DISCUSSION THREAD

**Discussion by beastyrabbit** [immich-app/immich#7158] [Feature Request]
### The feature

Subject: Feature Idea: Multi-IP Support for Home/Outside Access & Smarter Backups

Hey Immich Team,

Right now, the app uses the same connection setup whether I'm halfway around the world or chilling on my couch. While this is great for simplicity, it's not always the most efficient. When I'm at home, it would make a lot more sense to connect via my home network's local IP. This way, I can speed things up and save on mobile data.

So, here's what I'm thinking: Would it be possible to add a feature that lets the app check if I'm home and switch to my local IP for access and backups? And, to take it a step further, it would be awesome if the app could do backups specifically when I'm connected locally. This feels like it would save a lot of unnecessary data use and ensure backups happen super quickly.

I believe this could be a cool addition for not just me but a lot of us using the app who bounce between home and outside. Plus, it feels like a smart way to manage data and backups more efficiently.

Thanks for considering this idea. I'm really looking forward to seeing how the app evolves!

Cheers,
Tobias

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #308
